data/json/items/generic/dining_kitchen.json

```diff
@@ -652,7 +650,7 @@
     "type": "GENERIC",
     "category": "other",
     "id": "can_opener",
-    "name": { "str": "can opener" },
+    "name": { "str": "can opener", "str_pl": "can openers" },
```

Comment:
```suggestion
    "name": { "str": "can opener" },
```

Reply:
Sorry to ask but I am not used to working with github. This commit means that it was already changed in my version or do I need to make a new push with this change?

Reply:
Already changed.